### ISSUE TITLE
Sort the criteria in Simple and Advanced filter criteria list

### ISSUE THREAD

**Issue by sylvain-morin** [gbif/portal-feedback#4090]
Hello,

Everytime I showcase the GBIF portal, I'm wondering: why don't you sort the filter criteria list?

For exemple, I often demo the "Is in cluster" criteria, in the loooong advanced criteria list.
And I'm always scrolling up and down several times before finding it...

Any reason why it's not sorted?
If the non alphabetical order is wanted, maybe we could have an input to filter the filter criteria list?

For example, we could capture "cluster" and it will limit the criteria list to "Is in cluster".
Like the "search settings" input in chrome settings (chrome://settings/)

Thanks

**Comment by MortenHofft** [comment]
At least to begin with we tried to group the filters into categories. So e.g. `collectionCode`, `institutionCode`, `collectionName`, `institutionName` would go next to each other.

The simple options are the ones we either want to highlight are are used the most (I'm not sure how e.g. `CoordinateUncertainty` made it to the simple filters). And it felt wrong to reshuffle them when expanding with advanced (things changing location is always a bit confusing).

But I'm not happy with it either. It is very overwhelming. Search makes sense - that is also the approach used for the hosted portals. GBIF.org will be using the same interface in the future so we do not have to maintain two implementations
e.g. https://hp-theme.gbif-staging.org/occurrence/search

**Comment by sylvain-morin** [comment]
Great !
The More button opening the criteria list (with a filter input) of the hosted portal will be great.

Even if, when we are new to the portal, having all the criteria like in the current UI is faster to test/understand them (rather than selecting them one by one from the More popup). But that's a UX concerns :)

**Comment by MortenHofft** [comment]
Yes it is very tricky to find a good solution for all use cases I find. I was pondering having a little description for each of the filters. That might help for what you are describing.
Other ideas (and ideally mockups) are very welcomed.

**Comment by CecSve** [comment]
In the risk of making it overly complicated - an idea could be to have the description of the filters in the technical documentation so the description is updated with the documentation is updated? This could perhaps also make it easier in the future to add more filter options to the 'more' category.  

Another thing, I really like the examples of searches that are already a feature in the search!
